app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+from os import path
+
+
+class CopyCommandSyntaxError(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    command = command.split()
+
+    if command[0] != "cp":
+        raise CopyCommandSyntaxError("Copy command not found")
+
+    if len(command) != 3:
+        raise CopyCommandSyntaxError(
+            "Invalid copy command syntax.\n"
+            "For a successful copy "
+            "the command should look like this: "
+            "cp file.txt file-copy.txt"
+        )
+
+    file_to_copy = command[1]
```

Comment:
u could unpack ur command like this
`var1, var2, var3 = command` after len checking